test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
```

Comment:
Overall, great script. I wonder though, could we make it a bit more easy to grok if we followed the finish -> main -> functions convention?

That seems to be working well for some team members and also SEs.

A couple examples:
- https://github.com/conjur/jenkins-e2e-example/blob/master/e2e.sh
- https://github.com/conjurinc/appliance-uml/blob/master/smoketest.sh

reorg #nitpick :)